doc/user-manual/contribute/documentation.rst

```diff
@@ -209,18 +209,18 @@ shown again.
 File structure
 --------------
 
-Documentation literate files (`.lagda.*`) are type-checked as whole Agda files,
-as if all literate text was replaced by whitespace. Thus, **indentation** is
-interpreted globally.
+Documentation literate files (``.lagda.*``) are type-checked as whole Agda
+files, as if all literate text was replaced by whitespace. Thus, **indentation**
+is interpreted globally.
```

Comment:
The single backtick quotation is a common mistake that has been made in the RST files.  (RST is to blame to deviate from markdown here...) 

Do we need the line wrapping changes?  I think it is better to turn auto-wrapping of lines off in the editor to keep diffs minimal.

Reply:
Ah, okay. There's no auto-wrapping feature for RST files in VSCode as far as I am aware by the way.

Reply:
But am I correct that you operate with a lenient 80-character limit?

Reply:
Yes, lenient!